viewer.js

```diff
@@ -16,6 +16,8 @@ const params = getParams(window.location.search);
 let graph;
 let eventNamesVisible = params.names;
 let eventParamVisible = params.params;
+let ev_messages = {};
+
 
 const cboxShowParams = document.querySelector('#cbox-showparams');
 cboxShowParams.checked = eventParamVisible;
```

Comment:
consider adding a checkbox for this new feature ("Show message text", maybe?)

Reply:
Done